crowbar_framework/app/models/cinder_service.rb

```diff
@@ -205,6 +205,20 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
     # the VIP of the cluster to be setup
     allocate_virtual_ips_for_any_cluster_in_networks(controller_elements, vip_networks)
 
+    # Generate secrets uuid for libvirt rbd backend
+    proposal = ProposalObject.find_data_bag_item "crowbar/bc-cinder-default"
```

Comment:
As @bkutil proposed we can add option `applied: true` as solution for this. Please look on comment
https://github.com/crowbar/barclamp-cinder/pull/249/files#diff-13da7822dcdcc4488bbd522fc7e6f996R220